src/lib/arch/riscv/Instruction_decode.cc

```diff
@@ -40,25 +58,7 @@ Register csRegToRegister(unsigned int reg) {
  instructions tripping assertions.
  TODO remove once all extensions are supported*/
 void Instruction::invalidateIfNotImplemented() {
-  if (metadata.opcode >= Opcode::RISCV_ADD &&
-      metadata.opcode <= Opcode::RISCV_BNE)
-    return;
-  if (metadata.opcode >= Opcode::RISCV_DIV &&
-      metadata.opcode <= Opcode::RISCV_ECALL)
-    return;
-  if (metadata.opcode >= Opcode::RISCV_JAL &&
-      metadata.opcode <= Opcode::RISCV_SD)
-    return;
-  if (metadata.opcode >= Opcode::RISCV_SH &&
-      metadata.opcode <= Opcode::RISCV_SRAW)
-    return;
-  if (metadata.opcode >= Opcode::RISCV_SRL &&
-      metadata.opcode <= Opcode::RISCV_SW)
-    return;
-  if (metadata.opcode >= Opcode::RISCV_XOR &&
-      metadata.opcode <= Opcode::RISCV_XORI)
-    return;
-  if (metadata.opcode == Opcode::RISCV_FENCE) return;
+  return;
```

Comment:
Given this function now just does `return;`, can it be removed as the TODO comment states?

Reply:
Removed for cleanliness but may need to be re-implemented as Capstone updates and supports more extensions